toxcore/tox.api.h

```diff
@@ -286,7 +286,10 @@ const FILE_ID_LENGTH              = 32;
  */
 const MAX_FILENAME_LENGTH         = 255;
 
-
+/**
+ * Max size (in bytes) of names queried by toxcore
+ */
+const QUERY_MAX_NAME_SIZE       = 255;
```

Comment:
Two empty lines before separator block comment. Also, perhaps align `=` with the one above.